### ISSUE TITLE
White flashing with Dark mode when trigger check for updates

### ISSUE THREAD

**Issue by Girofox** [NexovaDev/UpdateHub#568]
### Issue
When switching to Apps section sometimes the whole App is in light mode while refreshing contents. This happens especially when new updates are searched or a manual "check for updates" is triggered. The title bar, fonts, sidebar entries etc. also disappears and changes shortly while refreshing.

<img width="400" alt="image" src="https://github.com/NexovaDev/UpdateHub/assets/43116061/976c40be-44ef-4465-a62d-b8b9284ef251">
<img width="400" alt="image" src="https://github.com/NexovaDev/UpdateHub/assets/43116061/1326ad60-9b73-4534-9db2-46e7981a18a7">

### How to trigger
While Dark mode toggle enabled go to section **Apps** and then click check for updates, if no issue repeat process some times with random interval.

### Suggestions
This seems to happen when the whole sidebar left is refreshing. Fonts, title bar etc. are maybe changing to a default change and even some hidden menus are appearing.

**Comment by KK-Designs** [comment]
Yeah it's refreshing the entire UI when you switch pages. I'm not entirely sure how to fix this but I'll try. I'll keep you updated here if I fix it
